File: app/src/main/java/com/sleticalboy/dailywork/weight/xrecycler/helper/PageDecorationLastJudge.java
package com.sleticalboy.dailywork.weight.xrecycler.helper;

/**
 * Created on 2016/11/15.
 * @author zhuguohui
 */
public interface PageDecorationLastJudge {
    /**
     * Is the last row in one page
     *
     * @param position
     * @return
     */
    boolean isLastRow(int position);

    /**
     * Is the last Colum in one row;
     *
     * @param position
     * @return
     */
    boolean isLastColumn(int position);

    boolean isLastLast(int position);
}

File: MainApp/src/main/java/com/binlee/learning/weight/xrecycler/helper/PageDecorationLastJudge.kt
package com.binlee.learning.weight.xrecycler.helper

/**
 * Created on 2016/11/15.
 * @author zhuguohui
 */
interface PageDecorationLastJudge {
  /**
   * Is the last row in one page
   *
   * @param position
   * @return
   */
  fun isLastRow(position: Int): Boolean

  /**
   * Is the last Colum in one row;
   *
   * @param position
   * @return
   */
  fun isLastColumn(position: Int): Boolean

  fun isLastLast(position: Int): Boolean
}
